Added documentation for Pluto XDMF support (#241)
neutrinoceros/yt_idefix: README.md

Now `yt.load` will be able to read Pluto/Idefix `.vtk` output files, as well as
Idefix `.dmp` dump files.

> Pluto `xdmf` (`.h5+.xmf`) files are supported from version 1.1.0. However, this feature requires the `h5py` external python module.

Reviewer: ```suggestion
Pluto [`XDMF`](https://www.xdmf.org/index.php/XDMF_Model_and_Format) files are supported from version 1.1.0. Note that this feature requires `h5py`.
```
